classes/conversion.php

```diff
@@ -318,7 +318,8 @@ private function create_conversion(\stored_file $file) : void {
         } catch (\dml_exception $e) {
             // If error is anything else but a duplicate insert, this is unexected,
             // so re-throw the error.
-            if (strpos($e->getMessage(), 'locasmarconv_pat_uix') === false && strpos($e->getMessage(), 'locasmarconv_con_uix') === false) {
+            if (strpos($e->getMessage(), 'locasmarconv_pat_uix') ===
+                    false && strpos($e->getMessage(), 'locasmarconv_con_uix') === false) {
```

Comment:
Can you please split this across multiple lines more cleanly? i.e. don't split on the `===` comparison.